suite-native/module-device-settings/src/screens/ContinueOnTrezorScreen.tsx

```diff
@@ -0,0 +1,57 @@
+import { useCallback } from 'react';
+import { Dimensions } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { selectDeviceModel } from '@suite-common/wallet-core';
+import { Box, Text } from '@suite-native/atoms';
+import { ConnectorImage, DeviceImage } from '@suite-native/device';
+import { Translation } from '@suite-native/intl';
+import { Screen, ScreenSubHeader } from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import TrezorConnect from '@trezor/connect';
+
+const SCREEN_HEIGHT = Dimensions.get('screen').height;
```

Comment:
super nitpick, ignore right away 😆 :  I would prefer using util `getScreenHeight`

Reply:
You can't import `/lib` folder, please fix the import `import { getScreenHeight } from '@trezor/env-utils';` 
Linter is also complaining about that so I hope it's possible to setup auto import feature to never use /lib folder.

Reply:
My bad, I haven't noticed.